src/NHibernate/Dialect/PostgreSQLDialect.cs

```diff
@@ -69,7 +69,9 @@ public PostgreSQLDialect()
 			RegisterFunction("mod", new SQLFunctionTemplate(NHibernateUtil.Int32, "((?1) % (?2))"));
 
 			RegisterFunction("sign", new StandardSQLFunction("sign", NHibernateUtil.Int32));
-			RegisterFunction("round", new RoundFunction());
+			RegisterFunction("round", new RoundFunction(false));
+			RegisterFunction("truncate", new RoundFunction(true));
+			RegisterFunction("trunc", new RoundFunction(true));
```

Comment:
As written [here](https://github.com/nhibernate/nhibernate-core/pull/1533/files#r172564339), I consider supporting dialect specific names (`trunc` here) as a bad practice. But since there is some disagreement about this, I will not fight against it and so I am still adding it here.